src/app/core/models/part.ts

```diff
@@ -24,6 +26,7 @@ export class Part {
   public expectedDuration?: number
   public executedAt: number
   public playedDuration: number
+  public autoNext: AutoNext | undefined
```

Comment:
Any reason for not writing it as `public autoNext?: AutoNext`?

Reply:
No. Is changed now.